src/main/java/com/teamc/fms/advice/GeneralControllerAdvice.java

```diff
@@ -10,13 +10,20 @@
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
 import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
+import java.util.HashMap;
+import java.util.Map;
+
 
 @ControllerAdvice
 @Slf4j
+@RestControllerAdvice
```

Comment:
I think no need for `@RestControllerAdvice` as `@ControllerAdvice` is sufficient